manage node as service by systemd (#1128)
witnet/witnet-rust: docs/get-started/installation/from-source.md

For more `witnet-rust` commands (`cli`, `wallet`, etc.) you can read the [Witnet-rust CLI documentation][CLI].

## Run as `systemd` service

> * Create the user `witnet` dedicated to running the node:

Reviewer: ```suggestion
* Create the `witnet` user dedicated to running the node:
```

```
sudo adduser witnet
```

> * Install latest version with the user `witnet` dedicated to running the node:

Reviewer: ```suggestion
* Install latest version with the `witnet` user dedicated to running the node:
```

```
sudo -u witnet ./systemd/runner.sh
```

> * Copy the file `witnet.service` in `/lib/systemd/system/`:

Reviewer: ```suggestion
* Copy the `witnet.service` file into `/lib/systemd/system/`:
```

```
sudo cp systemd/witnet.service /lib/systemd/system/witnet.service
```

> * Enable service:

Reviewer: ```suggestion
* Enable the service:
```

```
systemctl enable witnet.service
```

> * Start service:

Reviewer: ```suggestion
* Start the service:
```

```
systemctl start witnet.service
```

> * If for some reason the `witnet.service` file change, reload daemon:

Reviewer: ```suggestion
* If for some reason the `witnet.service` file changes, you need to reload the `systemd` daemon:
```

Author: outdated, already changed in the new version

```
systemctl daemon-reload
```

> * Restart service:

Reviewer: ```suggestion
* Restart the service:
```

Author: outdated, already added in the new version

```
systemctl restart witnet.service
```

> * See the logs of service:

Reviewer: ```suggestion
* See the logs of the service:
```

Author: outdated, already added in the new version
